### PR TITLE
fixes for code review

### PR DESCRIPTION
**Pull request by xdamman**
Thanks @dasilvacontin @sedubois for the code review, here are the fixes :-)

### CODE REVIEW

**Comment by dasilvacontin** [comment]
What about the fixed `imageHeight` I mentioned?

**Comment by xdamman** [comment]
For now we want to hard code that value. All avatars and logos should not be taller than 64px. We may change that in the future.

Or did you just mean to have one global constant?

**Comment by dasilvacontin** [comment]
> const svg = `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="${imageWidth}" height="64">
  <image width="${imageWidth}" height="${imageHeight}" xlink:href="data:${contentType};base64,${base64data}"/>
</svg>`

I'm pointing out that the svg size is the same as the contained image size (at least the element size), and you already have `imageWidth` and `imageHeight` to hold those values (`imageHeight` is always `64` currently), so it would make sense to use `imageHeight` also for the svg element's size, and not only for the contained image size.

And when you want to change the height, with another value or with the result of a function, you'd only have to change it in its assignment to `imageHeight`, instead of also having to change the hardcoded value in the svg string. 🚢

**Comment by xdamman** [comment]
Ah of course, good catch!
Fixing that.

File: server/controllers/banner.js [hunk 1]
```diff
@@ -76,19 +76,15 @@ module.exports = {
       request({url: imageUrl, encoding: null}, (e, r, data) => {
         const contentType = r.headers['content-type'];
 
-        var imageWidth, imageHeight;
-        if(tier === 'sponsor') {
+        const imageHeight = 64;
```
Comment (dasilvacontin) [inline server/controllers/banner.js]: Nice touch with the `const`!

**Comment by dasilvacontin** [comment]
LGMT~

File: server/controllers/banner.js [hunk 2]
```diff
@@ -76,19 +76,15 @@ module.exports = {
...
+        let imageWidth = 64;
```
Comment (asood123) [inline server/controllers/banner.js]: Why `let` vs `var`? I thought we were going to stay consistent with `var`.

File: server/controllers/banner.js (hunk 1 above)
Comment (xdamman) [inline server/controllers/banner.js]: `eslint` FTW!
eslint

**Pull request merged**